src/Slider/Slider.spec.js

```diff
@@ -217,6 +217,47 @@ describe('<Slider />', () => {
     assert.isAbove(wrapper.state().percent, previousPercent);
   });
 
+  it('simulates keydown event for the up arrow key on an x-reverse axis slider', () => {
```

Comment:
- What about nesting all those tests inside a context block?
```js
describe('keydown', () => {
```
- Looks like we have a lot of repeated code.
I'm wondering if we couldn't use a matrix of *input* / *ouput* to simplify it.